doc/QuickStart.md

```diff
@@ -54,14 +82,24 @@ connection.
 Here we clone the data-directory for two PBot instances, naming them after the
 IRC network they will connect to:
 
-    cp -r data freenode
-    cp -r data ircnet
+    $ su pbot
+    $ cd /home/pbot
+    $ git clone https://github.com/pragma-/pbot.git
+    $ cd pbot
+    $ cp -r data freenode
+    $ cp -r data ircnet
 
 Alternatively, you could name it after your bot's nickname:
 
-    cp -r data coolbot
+    $ cp -r data coolbot
+
+### Edit Registry *(pbot)*
 
-### Edit Registry
+	$ cd data
+	$ nano registry
+> or $ cd freenode && vi registry
+> or $ cd coolbot
+> ...
```

Comment:
Users are expected to know which text editor they prefer to use and how to use it.